api/src/pcapi/core/fraud/utils.py

```diff
@@ -6,6 +6,7 @@
 
 ACCEPTED_CHARS_FOR_NAMES = [" ", "-", ".", ",", "'", "’"]
 ACCEPTED_CHARS_FOR_CITY = [" ", "-", "'", "(", ")"]
+ACCEPTED_CHARS_FOR_ADDRESS = [" ", "-", ".", ",", "'", "’"]
```

Comment:
(détail / question de goût) est-ce que ce ne serait pas plus explicite comme ça ?

```suggestion
ACCEPTED_CHARS_FOR_ADDRESS = ACCEPTED_CHARS_FOR_NAMES
```

Reply:
Je vois l'idée, mais techniquement 2 besoins différents, je préfère séparer. 
Si on ajoute un caractère pour l'adresse, on ne veut pas nécessairement l'ajouter au noms

Reply:
ok